### PR TITLE
Adapt to PeleRad in PelePhysics

### PR DESCRIPTION
**Pull request by jrood-nrel**

### CODE REVIEW

**Comment by wjge** [comment]
GMRES didn't work for this chemistry previously.

File: Exec/RegTests/SootRadTest/kpDB/kpl_c2h4.dat
Comment (baperry2) [inline Exec/RegTests/SootRadTest/kpDB/kpl_c2h4.dat:1]: Could delete these and point to the versions in PelePhysics instead. They aren't heavy and hopefully we will eventually make them not needed, so leaving them here is also ok.

Comment (jrood-nrel) [reply Exec/RegTests/SootRadTest/kpDB/kpl_c2h4.dat:1]: I thought about that but a relative path that far away sounded like trouble and I didn't want to do anything fancy to try to account for it. If I have time I will just hard code them.

**Review by baperry2 — APPROVED** [review]

**Comment by baperry2** [comment]
It's important that we keep the triple flame test because it is one of the tutorials. Let me check and see if I can figure out why it is failing.

**Pull request merged**